### ISSUE TITLE
GuildCrafts repos store useful resources/guidance

### ISSUE THREAD

**Issue by tannerwelsh** [LearnersGuild/product-development#71]
tl;dr the https://github.com/GuildCrafts/web-development-js repo should contain helpful content for learning, in addition to the issues being used as goals.

To resolve this issue, define and create an initial file structure / schema for craft repos (i.e. repos in the GuildCrafts org).

This came about from conversation around goals, and noting that it is useful for learners to see things like a [set of skills to focus on](https://gist.github.com/tannerwelsh/bf845a198ddc5e566be4d13207d2bc4d#skills-to-focus-on).

If the craft repo contains useful resources/materials/guidance, then players can submit PRs against it to gradually improve this guide over time.

The craft repo can become a self-contained guide to the craft, with references, distilled wisdoms, and a list of goals (as issues) that can be converted into projects.
